### ISSUE TITLE
Remove `hand-tracking` from VRButton's `optionalFeatures` 

### ISSUE THREAD

**Issue by m-blix** [mrdoob/three.js#27698]
### Description

`VRButton` (and `XRButton`) has `hand-tracking` as part of the session's optional features.

```js
const sessionInit = { optionalFeatures: [ 'local-floor', 'bounded-floor', 'hand-tracking', 'layers' ] };
```
https://github.com/mrdoob/three.js/blob/master/examples/jsm/webxr/VRButton.js#L49

With the usage of `VRButton` in various examples that do not even use the WebXR hand tracking API (e.g. https://threejs.org/examples/webxr_vr_rollercoaster.html) this can trigger multiple successive permission prompts (1st for immersive session, second for hands) on certain platforms (e.g. Safari on visionOS).  With the Button's as is this annoyance can't be avoided.

### Solution

One option to avoid this is to remove `hand-tracking` from the `optionalFeatures` list, and instead provide a way to pass in additional features, similar to XRButton's createButton method.

Or, give access to VRButton.optionalFeatures = [...] for easy modification.

(And modify examples that need `hand-tracking` accordingly).

### Alternatives

Alternatively, this issue can be seen as responsibility of developer to copy and modify VRButton code, adjusting optionalFeatures (or requiredFeatures) to suit their needs.

**Comment by Mugen87** [comment]
>One option to avoid this is to remove hand-tracking from the optionalFeatures list, and instead provide a way to pass in additional features, similar to XRButton's createButton method.

I vote for that. We could apply the changes of #27030 to `VRButton` as well and then update the hand-tracking examples.

**Comment by Mugen87** [comment]
@m-blix Does https://github.com/mrdoob/three.js/pull/27699 look good to you?

**Comment by m-blix** [comment]
@Mugen87 Yes #27699 looks good 👍 

One other consideration, maybe for later or not at all as `optionalFeatures` is probably fine, but for examples that really require `hand-tracking` could have `hand-tracking` as part of session's `requiredFeatures` list.

**Comment by danrossi** [comment]
These sessions don't work in the visionOS simulator. How is it advised to detect for hand tracking support now it's trying to force it ?

**Comment by mrdoob** [comment]
> How is it advised to detect for hand tracking support now it's trying to force it ?

Can you rephrase the question? I don't understand it.

**Comment by danrossi** [comment]
These demos won't launch XR on VisionOS simulator as it seems there is no hand tracking to check controls and it requires it. I'll check again but it says "XR Not Allowed" So to support every device keep these properties optional is the advised method ? Making them required seems to only work on anything that supports it and there doesn't seem to be a method to detect what it supports. I'll do more digging. I figured out the visionOS video playback work around for others for now. Standard. I've done it before.

**Comment by mrdoob** [comment]
I'm Vision OS 1.1 beta they have started using transient pointers like the ones in Android WebXR. No need for hand tracking.

**Comment by Ledzz** [comment]
@mrdoob but these pointers exist only when clicking, so there's no way to make "hover" effects, am I right?

**Comment by mrdoob** [comment]
That's correct yeah...
